session/txn.go

```diff
@@ -381,8 +381,9 @@ func mergeToDirtyDB(dirtyDB *executor.DirtyDB, op dirtyTableOperation) {
 
 // txnFuture is a promise, which promises to return a txn in future.
 type txnFuture struct {
-	future oracle.Future
-	store  kv.Storage
+	future        oracle.Future
+	store         kv.Storage
+	lowResolution bool
```

Comment:
Where is it used?

Reply:
Oops. It was used earlier to detect update at commit time. Not useful anymore. Let me remove it. Thanks.